src/test/ui/in-band-lifetimes/ellided_lifetimes.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+#![deny(elided_lifetime_in_path)]
+struct Foo<'a> { x: &'a u32 }
+fn foo(x: &Foo) {
+//         ^^^ warning: hidden lifetime parameters are deprecated, try `Foo<'_>`
```

Comment:
This should be `//~^ ERROR: ...`

That is an annotation that indicates that we expect an error on the line above. The `.stderr` file will give more precise information.